src/routes/index.ts

```diff
@@ -460,6 +464,46 @@ router.route(config.UPDATE_BENEFICIAL_OWNER_TYPE_URL)
 
 router.post(config.UPDATE_BENEFICIAL_OWNER_TYPE_SUBMIT_URL, authentication, navigation.hasUpdatePresenter, updateBeneficialOwnerType.postSubmit);
 
+router.route(config.UPDATE_TRUSTS_SUBMISSION_INTERRUPT_URL)
+  .all(
+    isFeatureEnabled(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS),
+    authentication,
+    companyAuthentication,
+    navigation.hasUpdatePresenter,
+  )
+  .get(updateTrustsSubmissionInterrupt.get)
+  .post(updateTrustsSubmissionInterrupt.post);
+
+router.route(config.UPDATE_TRUSTS_TELL_US_ABOUT_IT_URL)
+  .all(
+    isFeatureEnabled(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS),
+    authentication,
+    companyAuthentication,
+    navigation.hasUpdatePresenter,
+  )
+  .get(updateTrustsTellUsAboutIt.get)
+  .post(updateTrustsTellUsAboutIt.post);
+
+router.route(config.UPDATE_TRUSTS_INDIVIDUALS_OR_ENTITIES_INVOLVED_URL)
+  .all(
+    isFeatureEnabled(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS),
+    authentication,
+    companyAuthentication,
+    navigation.hasUpdatePresenter,
+  )
+  .get(updateTrustsIndividualsOrEntitiesInvolved.get)
+  .post(updateTrustsIndividualsOrEntitiesInvolved.post);
+
+router.route(config.UPDATE_TRUSTS_ASSOCIATED_WITH_THE_OVERSEAS_ENTITY_URL)
+  .all(
+    isFeatureEnabled(config.FEATURE_FLAG_ENABLE_UPDATE_TRUSTS),
+    authentication,
+    companyAuthentication,
+    navigation.hasUpdatePresenter,
```

Comment:
just a note and thinking out loud - we're going to have to remember to update the navigation.hasUpdatePresenter nav checks on each of these when we get to individual tickets and start adding stuff to appData.